src/main.js

```diff
@@ -136,6 +157,7 @@ async function run(shouldIncludeVerifiedAccounts) {
     elShouldIncludeVerified.disabled = true;
```

Comment:
While you're at it, if it's not too much trouble, can you combine this line with the one above and make it simply - `getElementByClass('.iu_include-verified-checkbox').disabled = true` ?

Not sure why I wrote it this way originally.

Reply:
Sure.
I will close this PR and submit a new one.
Thanks for your feedback.